spec/defines/resource_upstream_spec.rb

```diff
@@ -36,6 +38,7 @@
         ]
       end
 
+      # rubocop:disable RSpec/MultipleMemoizedHelpers
```

Comment:
Maybe that will make sense to disable it globally?

Reply:
I was thinking about it but I have mixed feelings: on one side it is a smell that really should be avoided, but at the same time in acceptance tests it is something ugly for which I can't find a clean solution.

Disabling these checks here is the easy way to move on quickly without blocking everything, not something I expect to see generalized.